### PR TITLE
minify manifest json

### PR DESCRIPTION
**Pull request by Gsealy**
fix: #28

### CODE REVIEW

File: images/encryption/encryption.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ func cryptChildren(ctx context.Context, cs content.Store, desc ocispec.Descripto
 			Layers: newLayers,
 		}
 
-		mb, err := json.MarshalIndent(newManifest, "", "   ")
```
Comment (lumjjb) [inline images/encryption/encryption.go:320]: I think there are couple other instances of this, could you change them at the same time? Thanks!
```
cmd/ctr/commands/commands.go:181:	b, err := json.MarshalIndent(x, "", "    ")
images/encryption/encryption.go:320:		mb, err := json.MarshalIndent(newManifest, "", "   ")
images/encryption/encryption.go:400:		mb, err := json.MarshalIndent(newIndex, "", "   ")
```

Comment (Gsealy) [reply images/encryption/encryption.go:320]: done!

File: cmd/ctr/commands/commands.go [hunk 2]
```diff
@@ -178,7 +178,7 @@ func LabelArgs(labelStrings []string) map[string]string {
 
 // PrintAsJSON prints input in JSON format
 func PrintAsJSON(x interface{}) {
-	b, err := json.MarshalIndent(x, "", "    ")
```
Comment (stefanberger) [inline cmd/ctr/commands/commands.go:181]: I would just leave it as is. This whole file is a copy if the corresponding file from containerd: https://github.com/containerd/containerd/blob/v1.5.0-beta.0/cmd/ctr/commands/commands.go#L206

Comment (Gsealy) [reply cmd/ctr/commands/commands.go:181]: we can compress it when save or output OCI image tar? not affect input json

**Comment by Gsealy** [comment]
closed.

**Pull request closed without merging**